### PR TITLE
DG - Updated Appendix F for Corrupted files and changing of names

### PR DESCRIPTION
**Pull request by yAOwzers**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1210,4 +1210,17 @@ i. Click on the folder that the jar file had been saved in.
 ii. Select both the 'tasks.txt' and 'notebooks.txt'.
 iii. Delete both files.  
 iv. Restart the application by double-clicking the jar file and running Zer0Note.  
-Expected: The Command Line Interface should launch with a welcome note from Zer0Note as shown in Appendix F, 1.1.  
+Expected: The Command Line Interface should launch with a welcome note from Zer0Note as shown in Appendix F, 1.1. 
+
+3.2 Dealing with corrupted name file  
+i. Click on the folder that the jar file had been saved in.  
+ii. Enter the 'src', then 'main', 'resources' and then 'txt' folder.
```
Comment (franceneee) [inline docs/DeveloperGuide.md:1217]: wait, I'm not sure if the user has access to src... o.o I thought they just download the jar?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1210,4 +1210,17 @@ i. Click on the folder that the jar file had been saved in.
...
+
+3.2 Dealing with corrupted name file  
+i. Click on the folder that the jar file had been saved in.  
+ii. Enter the 'src', then 'main', 'resources' and then 'txt' folder.  
+iii. Delete the file named 'nameOfUser.txt'  
+iv. Restart the application by double-clicking the jar file. Re-enter the name of user.  
+
+3.3 Changing of saved name  
+i. Click on the folder that the jar file had been saved in.  
+ii. Enter the 'src', then 'main', 'resources' and then 'txt' folder.
```
Comment (franceneee) [inline docs/DeveloperGuide.md:1223]: Same as the previous

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yAOwzers) [reply docs/DeveloperGuide.md:1217]: Oh wait they arent doing this?

image

Comment (franceneee) [reply docs/DeveloperGuide.md:1217]: OH RIGHT THIS IS THE DG AND NOT UG SORRY

**Review by franceneee — APPROVED** [review]
Okay! but there are merge conflicts hehe. xD

**Pull request merged**